src/templates/program-page.js

```diff
@@ -345,7 +345,7 @@ const ProgramPage = ({ data, location }) => {
           <section className="row row-with-vspace site-content">
             <div className="col-md-12 content-area">
               {renderAdmissionRequirements()}
-              {renderTalkToStudent()}
+              {/****renderTalkToStudent()****/}
```

Comment:
The `renderTalkToStudent` function starting on line 47 should also be commented out - otherwise, a warning will appear saying it's not being used.